docs/content/preview/reference/configuration/yb-tserver.md

```diff
@@ -1326,6 +1326,36 @@ Valid values are `-1` (unlimited), `integer` (in kilobytes), `nMB` (in megabytes
 
 Default: `1GB`
 
+## Catalog flags
+
+##### ysql_catalog_preload_additional_table_list
+
+Specifies the names of catalog tables (such as pg_operator, pg_proc, and pg_amop) to be preloaded by PostgreSQL backend processes. This flag reduces latency of first execution of a particular statement on a connection.
+
+Default: `""`
+
+If [ysql_catalog_preload_additional_tables](#ysql-catalog-preload-additional-tables) is also specified, the union of the above specified catalog tables and pg_am, pg_amproc, pg_cast, and pg_tablespace is preloaded.
+
+##### ysql_catalog_preload_additional_tables
+
+When enabled, PG backend processes preloads the catalog tables: `pg_am`, `pg_amproc`, `pg_cast`, and `pg_tablespace`. This flag reduces latency of first execution of a particular statement on a connection.
+
+Default: false
```

Comment:
Should the default values be in backticks (e.g. `false`)? This seems to be the convention based on the [existing flag docs](https://docs.yugabyte.com/preview/reference/configuration/yb-tserver/#stop-logging-if-full-disk).